test/framework/cloudstack.go

```diff
@@ -0,0 +1,141 @@
+package framework
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+)
+
+const (
+	cloudstackDomainVar                = "T_CLOUDSTACK_DOMAIN"
+	cloudstackZoneVar                  = "T_CLOUDSTACK_ZONE"
```

Comment:
Should we add zone1, zone2 to support multiple-zone? Similar for network. 

not a block for this PR, we can create a issue/feature so that we remember to do it.

Reply:
Agreed, will create an issue